### DISCUSSION TITLE
Add files to cdnjs

### DISCUSSION THREAD

**Discussion by tom177y** [cdnjs/cdnjs#14191] [General] [answered]
Hi, I have some font files to be hosted in cdnjs because the current SaaS provider has a following cross-origin configuration:
1) Referral Policy: same-origin
2) content-security-policy: font-src 'self' cdnjs.cloudflare.com

The SaaS application allows custom styling to be added but when attempt to load the font files externally, the cross-origin (font) resources were blocked.  Chrome showed (blocked:csp) status.

According to https://github.com/cdnjs/packages/blob/master/CONTRIBUTING.md, a GitHub repository requirement is normally 200 stars before can be added to cdnjs.  

Any suggestion? Tqvm.

**Comment by MattIPv4** [comment, marked as answer]
Hey!

cdnjs doesn't host arbitrary files from users, but rather we host established libraries (e.g. Bootstrap).

If the font you're looking to be added is published as a library on NPM (or tagged with versions on Git), and meets our criteria for notability, please open a PR or issue in cdnjs/packages to add it :)

**Comment by tom177y** [comment]
Hi Matt, thanks for the reply.  I am of the same opinion too. Hosting such packages from users will defeat real intent of cdnjs.  I will find alternative solution.

On separate note, I am trying to find out more info on https://cc-cdn.com.  The domain is owned by Cloudflare and it too publishes libraries.   Would like to know if it's possible to publish the font files here instead.  If yes, how do I go about doing it.

Thank you.

**Reply by MattIPv4** [reply]
I am not familiar with that domain, though it looks to be an S3 bucket. I doubt it's for public use.

if you're just looking to host some files, there are plenty of ways to do it, paid or free. A static site host like GitHub Pages or Cloudflare Pages can host a website and files for said website, or you could use a dedicated storage solution like AWS S3, Cloudflare R2, Backblaze B2, or DigitalOcean Spaces.

**Reply by tom177y** [reply]
Hi Matt, reason why I ended exploring cdnjs.com and cc-cdn.com was because the current setting for **content-security-policy** contains these domain entries, which I don't have control over. It seems like best option is to request an additional entry (S3 bucket) to be added to the directive.
